test/parallel/test-timers-api-refs.js

```diff
@@ -1,20 +1,24 @@
 'use strict';
 const common = require('../common');
-const assert = require('assert');
+const timers = require('timers');
```

Comment:
(Also: Yeah, it works. Test fails with a `ReferenceError` in Node 5.9.1, as it should, and passes with your changes in `lib/timers.js`.)